systemd/ngrok_displayer.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
If we plan on using ngrok in the future (ex: more remote comps(?)) I think it would be a good idea to leave a comment here explaining what this script is for and how it should be used